### PR TITLE
prep-javascript-arrays exercise - Adrian Guzman

### PR DESCRIPTION
**Pull request by adrian-guzman-301**
prep javascript arrays

### CODE REVIEW

File: prep-javascript-arrays/main.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  'css',
+  'javascript',
+  'react',
+  'node',
+  'postgres'
+];
+console.log('languages array', languages);
+const firstElement = 'html';
+console.log('the first element of the array is', firstElement);
+const thirdElement = 'javascript';
```
Comment (thebearingedge) [inline prep-javascript-arrays/main.js:12]: Hi Adrian, you can access elements of an array by their index.

```js
var firstElement = languages[0];
var thirdElement = languages[2];
```

File: prep-javascript-arrays/main.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+console.log('the third element of the array is', thirdElement);
+var length = 6;
+console.log('the length of the array is', length);
```
Comment (thebearingedge) [inline prep-javascript-arrays/main.js:15]: You can get the length of an array with its `length` property.

```js
var length = languages.length;
```

File: prep-javascript-arrays/main.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+const lastElement = 'postgres';
+console.log('the last element of the array is', lastElement);
```
Comment (thebearingedge) [inline prep-javascript-arrays/main.js:17]: For this part the goal is to subtract `1` from `length` and use the result as the index into the `languages` array.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: prep-javascript-arrays/main.js (hunk 3 above)
Comment (adrian-guzman-301) [reply prep-javascript-arrays/main.js]: prep javascript arrays resubmission

**Pull request merged**